src/utils/execWithOutput.js

```diff
@@ -11,16 +11,14 @@ const { exec } = require('@actions/exec')
  * @param {{cwd?: string}} options
  * @returns Promise<string>
  */
-async function execWithOutput(cmd, args, { cwd } = {}) {
+async function execWithOutput(cmd, args, { cwd, ...options } = {}) {
   let output = ''
   let errorOutput = ''
 
   const stdoutDecoder = new StringDecoder('utf8')
   const stderrDecoder = new StringDecoder('utf8')
 
-  const options = {
-    silent: false,
-  }
+  options.silent = false
```

Comment:
minor: can we implement this differently so if for whatever reason the caller specifies that silent should be true, we don't override it?